Doc/using/mac.rst

```diff
@@ -2,13 +2,13 @@
 .. _using-on-mac:
 
 ***************************
-Using Python on a Macintosh
+Using Python on a Mac
```

Comment:
Fix lines before and after this line.

Is Mac an official name of all computers running macOS?

Reply:
The number of `*`s should match the length of the title.

Reply:
According to Wikipedia "Mac"  is a correct name for modern Apple's computers.